public/js/components/submit-button.jsx

```diff
@@ -6,16 +6,21 @@ export default class SubmitButton extends Component {
 
   static propTypes = {
     isDisabled: PropTypes.bool,
+    modifier: PropTypes.string.isRequired,
```

Comment:
It doesn't but then it's still required. If defaults were changed this would still stand.